hello.txt

```diff
@@ -1,3 +1,60 @@
 hello this is a message from brackets...
 
-Ok. I checked out feature3 branch before writing this. Are we really only on 3?
\ No newline at end of file
+Ok. I checked out feature3 branch before writing this. Are we really only on 3?
+
+Sprucing it up with a few Hamilton lyrics:
+
+ELIZA:
+Un deux trois quatre

```

Comment:
Ha ha ha... Why don't you write to Lin Manuel Miranda and ask him for the translation!?